extensions/sdk/src/Rally.ts

```diff
@@ -301,8 +301,14 @@ export class Rally {
       await browser.tabs.reload();
     } else {
       // Otherwise, open the website.
+      const url = new URL(this._options.rallySite);
+      const attribution = this.getAttributionCodes();
+      ["source", "medium", "campaign", "term", "content"].forEach((key) => {
+        attribution[key] = url.searchParams.set(`utm_${key}`, key);
```

Comment:
shouldn't this be ```url.searchParams.set(`utm_${key}`, attribution[key])```? without the equal sign?

also maybe we should guard against setting the param if the key doesn't exist?